unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -73,6 +73,39 @@ export function ComboboxValuesTest() {
       });
     });
 
+    describe('when calling clearAll without values', () => {
+      function addChildToCombobox() {
+        comboboxValues.element.appendChild($$('div').el);
+      }
+
+      beforeEach(() => {
+        initializeComponent(['updateAccessibilityAttributes']);
+        addChildToCombobox();
+        comboboxValues.clearValues();
+      });
+
+      it('should log analytics', () => {
```

Comment:
The test description mentions `analytics`, but the implementation seems focused on the accessibility attributes?

Reply:
Woops, I have no excuse here, the only way I know to describe this mistake is as a brainfart.